lib/functions.sh

```diff
@@ -155,6 +155,16 @@ setHostname() {
 	fi
 }
 
+# blaze memory cap should be approximately a quarter of the system memory
+# the memory cap will be applied to both the java heap size and db clock cache
+setBlazeMemoryCap() {
+	if [ -z "$BLAZE_MEMORY_CAP" ]; then
+	   system_memory=$(grep MemTotal /proc/meminfo | grep -Po '\d+');
+	   system_memory_in_mb=$(("$system_memory"/1024));
```

Comment:
Depending on the /proc file system is always a bit tricky and potentially not portable. How about the following:
```suggestion
	   system_memory_in_mb=$(free -m | grep 'Mem:' | awk '{print $2}');
```